wire-ios-data-model/Source/MLS/OneOnOne/OneOnOneResolver.swift

```diff
@@ -107,7 +107,19 @@ public final class OneOnOneResolver: OneOnOneResolverInterface {
                 else {
                     return
                 }
+
+                let selfUser = ZMUser.selfUser(in: context)
+
+                if !conversation.isForcedReadOnly {
+                    if !selfUser.supportedProtocols.contains(.mls) {
+                        conversation.appendMLSMigrationMLSNotSupportedForSelfUser(user: selfUser, at: .now)
```

Comment:
Maybe the `at: .now()` can be moved inside the `func` one level deeper, because here from the outside it's kind of logical that in the moment of calling to append a message it should be appended now.

Reply:
Done!